app/javascript/controllers/application.js

```diff
@@ -20,9 +20,24 @@ Turbo.setConfirmMethod((message) => {
       document.getElementById("turbo-confirm-body").innerHTML = body;
     }
 
+    if (acceptClass) {
+      document.getElementById("turbo-confirm-accept").className += acceptClass
+    }
+
     if (accept) {
       document.getElementById("turbo-confirm-accept").innerHTML = accept;
     }
+
+    if (reject) {
+      const button = document.createElement("button")
+      button.setAttribute("id", "turbo-confirm-reject")
+      button.setAttribute("class", "w-full text-red-600 rounded-xl text-center p-[10px] border mt-2")
+      button.setAttribute("value", "reject")
+      button.innerHTML = reject
+
+      document.getElementById("turbo-confirm-dialog-form").appendChild(button)
```

Comment:
I think we could probably move this to the `_confirm_modal` partial and then toggle classes as needed here.

Reply:
I just didn't want to modify the _confirm_modal partial. I'll do that now instead. So we just toggle visibility based on args.